### PR TITLE
Floating-point consensus acceptance criteria

### LINKED ISSUE #765: Add acceptance criteria to the floating-point-consensus spec
As the development work has started on the floating-point-consensus feature we need to update the spec to include acceptance critera.

**Requirements**
For people that were not in the research offsite and for future reference we need to have acceptance criteria addded to the following spec:

https://github.com/vegaprotocol/specs-internal/blob/master/protocol/0065-floating-point-consensus.md

### PR DESCRIPTION
**Pull request by witgaw**
Added acceptance criteria + clarification on which events trigger which (re-)calculations.

Closes https://github.com/vegaprotocol/specs-internal/issues/765

### CODE REVIEW

File: protocol/0065-floating-point-consensus.md [hunk 1]
```diff
@@ -80,3 +81,15 @@ This section outlines floating-point quantities `vega` currently relies on:
...
 - [`ProbabilityOfTrading(currentP, orderP *num.Uint, minP, maxP, yFrac num.Decimal, isBid, applyMinMax bool) num.Decimal`](https://github.com/vegaprotocol/vega/blob/4be994751b0012b0904e37ad2b0d1540d24abb5e/risk/model.go#L26) - calculates probability of trading at a specified projection horizon of the specified order given current price and min/max bracket, called from [supplied liquidity engine](https://github.com/vegaprotocol/vega/blob/4be994751b0012b0904e37ad2b0d1540d24abb5e/liquidity/engine.go#L34) each time new liquidity provision orders are deployed.
+
+## Acceptance criteria
+
+- Default risk-factors are both equal to 1.
+- Default probability of trading is 0.005.
+- Default price monitoring bounds are 10% up and 10% down.
```
Comment (davidsiska-vega) [inline protocol/0065-floating-point-consensus.md]: I think all the defaults should be phrased under one topic as one story that the QA can then test. So something like 
1) a market is proposed, 
2) verify that a fresh market is initialised with the correct defaults (list them)
3) verify that the defaults get updated with the calculated values.

File: protocol/0065-floating-point-consensus.md [hunk 2]
```diff
@@ -80,3 +81,15 @@ This section outlines floating-point quantities `vega` currently relies on:
...
+- Market enactment triggers state variable recalculation (verify that default values are no longer used).
```
Comment (davidsiska-vega) [inline protocol/0065-floating-point-consensus.md]: Could we have this as another topic / story for QA?

**Review by davidsiska-vega — CHANGES_REQUESTED** [review]
I think we should expand all the individual acceptance criteria into stories of a few more steps so that QA can use them to create their tests. 

Happy to help flesh those out on a call @witgaw (at the moment I am a bit hazy on all the details).

**Comment by edd** [comment]
~I'm going to deal with the merge conflict after merging #799~

Edit: Done

File: protocol/0065-floating-point-consensus.md (hunk 1 above)
Comment (witgaw) [reply protocol/0065-floating-point-consensus.md]: Added the scenarios sketches now.

File: protocol/0065-floating-point-consensus.md (hunk 2 above)
Comment (witgaw) [reply protocol/0065-floating-point-consensus.md]: Added the scenarios sketches now.

**Review by davidsiska-vega — DISMISSED** [review]
👍

**Pull request merged**